speculos/mcu/seproxyhal.py

```diff
@@ -329,10 +330,10 @@ def can_read(self, screen: DisplayNotifier):
                     # Publish the new screenshot, we'll upload its associated events shortly
                     screen.display.gl.update_public_screenshot()
 
-                if screen.display.model != "stax" and screen.display.screen_update():
+                if not isinstance(screen.display.gl, NBGL) and screen.display.screen_update():
```

Comment:
Could be set as a `self.use_bagl` at `__init__`?

Reply:
Yes, agree